### DISCUSSION TITLE
latex模版的版式存在页脚太高、页眉太低、宽度太窄的问题

### DISCUSSION THREAD

**Discussion by jiaqiwang969** [sjtug/SJTUThesis#844] [提问]
您好！

目前还发现一个问题，这与之前课题组和其他课题组的博士论文不太相符：

即存在：
latex模版的版式存在页脚太高、页眉太低的问题。

这导致一页容纳的内容过少。

请问，如何调整页脚、页眉的高度？

![image](https://user-images.githubusercontent.com/76413247/190912045-417ea4b7-8f22-4df7-a7c0-14d071f76fcf.png)
![image](https://user-images.githubusercontent.com/76413247/190912143-52b377a6-4e61-4a9e-916f-513fc1c96e3e.png)

**Comment by LogCreative** [comment]
见 https://github.com/sjtug/SJTUThesis/issues/652#issuecomment-1002606696 。只要符合规范即可，Word 版只是参考实现。

如果需要设置页边距，调整下面的代码

https://github.com/sjtug/SJTUThesis/blob/700093d999dc6fe212aa280d04aee6c120470f3b/texmf/tex/latex/sjtuthesis/sjtuthesis-graduate.ltx#L26-L34

在调整时请注意符合相关规范。

**Reply by jiaqiwang969** [reply]
谢谢！

**Comment by jiaqiwang969** [comment]
<img src="https://cdn.mathpix.com/snip/images/vC5yw4P2hazqdNBfHBEW34rnMJSkF4K_yif-OIPG78M.original.fullsize.png" />
宽度同样太窄
